tests/integration-tests/tests/performance_tests/test_osu.py

```diff
@@ -0,0 +1,267 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+import re
+
+import boto3
+import pytest
+from assertpy import assert_that
+from remote_command_executor import RemoteCommandExecutor
+
+from tests.common.assertions import assert_no_errors_in_logs
+from tests.common.osu_common import run_individual_osu_benchmark
+from tests.common.utils import fetch_instance_slots, get_installed_parallelcluster_version, run_system_analyzer
+
+
+@pytest.mark.usefixtures("serial_execution_by_instance")
+def test_osu(
+    os,
+    region,
+    scheduler,
+    instance,
+    pcluster_config_reader,
+    clusters_factory,
+    test_datadir,
+    architecture,
+    network_interfaces_count,
+    mpi_variants,
+    scheduler_commands_factory,
+    request,
+):
+    # We collected OSU benchmarks results for c5n.18xlarge only.
+    osu_benchmarks_instances = ["c5n.18xlarge"]
+
+    if instance not in osu_benchmarks_instances:
+        raise Exception(f"OSU benchmarks can't be run on instance {instance}.")
```

Comment:
Let's add to the log line what are the supported instances. In this way if a technician uses the wrong instance type he/she can know immediately what should be used.